webviz_subsurface/containers/_summary.py

```diff
@@ -0,0 +1,670 @@
+                                          for i in self.smry_vector_columns],
+                                 value=self.smry_vector_columns[0]),
+                    html.Div([
+                        dcc.Checklist(
+                            id=self.chlst,
+                            options=[{
+                                'label': 'Fieldgains',
+                                'value': 'show_fieldgains'
+                            }],
+                            labelStyle={'display': 'inline-block'},
+                            value=[],
+                        ),
+                        html.Div(id=self.show_ens_selectors, children=[
+                            dcc.Dropdown(
+                                id=self.dropdown_iorens_id,
+                                placeholder="Base case",
+                                options=[{'label': i, 'value': i}
+                                         for i in self.base_ensembles]
+                            ),
+                            dcc.Dropdown(
+                                id=self.dropdown_refens_id,
+                                placeholder="Select ensembles",
+                                options=[{'label': i, 'value': i}
+                                         for i in self.delta_ensembles],
+                                multi=True,
+                            ),
+                        ], style={'display': 'none'}),
+                    ]),
+                ], style={'width': '20%', "float": "left"}),
+
+                html.Div([
+                    dcc.Tabs(id=self.tab_id, value='summary_data', children=[
+                        dcc.Tab(label='Realizations', value='summary_data'),
+                        dcc.Tab(label='Statistics', value='summary_stats'),
+                    ]),
+                    html.Div(id='tabs-content'),
+                    html.Div(id=self.chart_id)
+                ], style={'width': '80%', "float": "right"}),
+
+            ]),
+        ])
+
+
+# =============================================================================
+# Callbacks
+# =============================================================================
+
+    def set_callbacks(self, app):
+
+        """ define callbacks for SUmmary-container => dash functionality"""
+
+        @app.callback(Output(self.show_ens_selectors, 'style'),
+                      [Input(self.chlst, 'value')])
+        def func_show_ens_selectors(chlst: list):
+            """callback to update the styling of div that includes the ensemble
+            selectors. The styling switches to hiden when fieldgains is not
+            selected.
+
+            Input:
+                checklist values: list of strings = selected options
+            Output:
+                html.Div(...styling): dictionary describing styling
+            """
+
+            if 'show_fieldgains' in chlst:
+                return {}
+
+            else:
+                return {'display': 'none'}
```

Comment:
These lines can be replaced with
`return {} if 'show_fieldgains' in chlst else {'display': 'none'}`

Reply:
Changed